packages/now-build-utils/.gitignore

```diff
@@ -1,3 +1,4 @@
+?.js
```

Comment:
What does this mean?

![double rainbow](https://media1.tenor.com/images/04dd032f5547f668c7d6147997bf6803/tenor.gif?itemid=11850694)

Reply:
I have ephemeral development/testing files like `d.js`. This way they don't get committed by accident.